tests/devices/unit_tests/test_dcm.py

```diff
@@ -0,0 +1,58 @@
+from typing import AsyncIterable
+
+import pytest
+from ophyd_async.core import (
+    DeviceCollector,
+)
+
+from dodal.devices.dcm import DCM
+from tests.conftest import MOCK_DAQ_CONFIG_PATH
+
+
+@pytest.fixture
+async def dcm() -> DCM:
+    async with DeviceCollector(sim=True):
+        dcm = DCM("DCM-01", name="dcm", daq_configuration_path=MOCK_DAQ_CONFIG_PATH)
+    return dcm
+
+
+@pytest.fixture
+async def staged_dcm(dcm: DCM) -> AsyncIterable[DCM]:
+    await dcm.stage()
+    yield dcm
+    await dcm.unstage()
+
+
+def test_lookup_table_paths_passed(dcm: DCM):
+    assert (
+        dcm.dcm_pitch_converter_lookup_table_path
+        == MOCK_DAQ_CONFIG_PATH + "/lookup/BeamLineEnergy_DCM_Pitch_converter.txt"
+    )
+    assert (
+        dcm.dcm_roll_converter_lookup_table_path
+        == MOCK_DAQ_CONFIG_PATH + "/lookup/BeamLineEnergy_DCM_Roll_converter.txt"
+    )
+
+
+async def test_offset_decoded(dcm: DCM):
+    assert dcm.fixed_offset_mm == 25.6
+
+
+@pytest.mark.parametrize(
+    "key",
+    [
+        "dcm-backplate_temp",
+        "dcm-bragg_in_degrees",
+        "dcm-energy_in_kev",
+        "dcm-offset_in_mm",
+    ],
+)
+async def test_read_and_describe_includes(
```

Comment:
Should: This works fine with an unstaged `DCM`. What is staging the `DCM` doing? If nothing can we remove the `staged_dcm` fixture? If something can we test that thing?

Reply:
My mistake, I was confusing them with `StandardDetector`s, which should be staged before being read